packages/peregrine/lib/targets/peregrine-declare.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * These targets are available for interception to modules which depend on `@magento/peregrine`.
+ *
+ * Their implementations are found in `./peregrine-intercept.js`.
+ *
+ * @module Peregrine/Targets
+ */
+module.exports = targets => {
+    targets.declare({
+        /**
+         * @callback talonIntercept
+         * @param {Peregrine/Targets.TalonWrapperConfig} talons - Registry of talon namespaces, talons, and Sets of interceptors
+         * @returns {undefined} - Interceptors of `talons` should add to the
+         * sets on passed TalonWrapperConfig instance. Any returned value will
+         * be ignored.
+         */
+
+        /**
+         * Collects requests to intercept and "wrap" individual Peregrine
+         * talons in decorator functions. Use it to add your own code to run
+         * when Peregrine talons are invoked, and/or to modify the behavior and
+         * output of those talons.
+         *
+         * This target is a convenience wrapper around the
+         * `@magento/pwa-buildpack` target `transformModules`. That target uses
```

Comment:
```suggestion
         * `@magento/peregrine` target `transformModules`. That target uses
```

Reply:
I'm not sure I understand. `transformModules` is a Buildpack target, not a Peregrine target!

Reply:
Oh I think I was just looking for references like the one you fixed earlier. This is fine.